librz/main/rz-find.c

```diff
@@ -633,6 +634,15 @@ RZ_API int rz_main_rz_find(int argc, const char **argv) {
 			return 1;
 		}
 		rzfind_open(&ro, file);
+
+		if (ro.exec_command) {
```

Comment:
this should be probably executed within `rzfind_open`

Reply:
Okayy